### ISSUE TITLE
Broken link in "Standalone applications"

### ISSUE THREAD

**Issue by glaz666** [oreilly/couchdb-guide#330]
The link to working calendar application is broken (http://jchrisa.net/cal/_design/cal/index.html) in the following chapter: http://guide.couchdb.org/editions/1/en/standalone.html#wild

**Comment by silverknight** [comment]
This link exists and is equally broken in the "draft" version of the book as well.  :)

**Comment by glaz666** [comment]
Yes. More over, I can see that code of Sofa application gone much further than the state of the book. Better either provide working examples of application in separate github repo or put complete listings in the book
